extensions/amp-next-page/0.1/test/test-amp-next-page.js

```diff
@@ -177,6 +177,37 @@ env => {
             'i-amphtml-next-page-hidden'))
         .to.be.true;
   });
+
+  it('removes amp-analytics tags from child documents', function* () {
```

Comment:
@aghassemi @peterjosling This test is flaky, and is breaking `master`. Please skip / fix.

/cc @zhouyx, who is this week's build cop.

```
FAILED TESTS:
  amp-next-page component
     
      ● removes amp-analytics tags from child documents
        Chrome 66.0.3359 (Mac OS X 10.13.3)
      TypeError: Cannot read property 'returnValue' of null
          at _callee5$ (/var/folders/nv/ycwrjg_j32gdz6s542xlxy0m0037zm/T/extensions/amp-next-page/0.1/test/test-amp-next-page.js:204:51 <- /var/folders/nv/ycwrjg_j32gdz6s542xlxy0m0037zm/T/d6180c0ce24ad620fb39441b3b820153.browserify:1597:54)
          at tryCatch (/var/folders/nv/ycwrjg_j32gdz6s542xlxy0m0037zm/T/node_modules/regenerator-runtime/runtime.js:65:0 <- /var/folders/nv/ycwrjg_j32gdz6s542xlxy0m0037zm/T/d6180c0ce24ad620fb39441b3b820153.browserify:20162:40)
          at Generator.invoke [as _invoke] (/var/folders/nv/ycwrjg_j32gdz6s542xlxy0m0037zm/T/node_modules/regenerator-runtime/runtime.js:303:0 <- /var/folders/nv/ycwrjg_j32gdz6s542xlxy0m0037zm/T/d6180c0ce24ad620fb39441b3b820153.browserify:20400:22)
          at Generator.prototype.(anonymous function) [as next] (/var/folders/nv/ycwrjg_j32gdz6s542xlxy0m0037zm/T/node_modules/regenerator-runtime/runtime.js:117:0 <- /var/folders/nv/ycwrjg_j32gdz6s542xlxy0m0037zm/T/d6180c0ce24ad620fb39441b3b820153.browserify:20214:21)
          at step (/var/folders/nv/ycwrjg_j32gdz6s542xlxy0m0037zm/T/testing/yield.js:48:33 <- /var/folders/nv/ycwrjg_j32gdz6s542xlxy0m0037zm/T/d6180c0ce24ad620fb39441b3b820153.browserify:76712:33)
          at _next (/var/folders/nv/ycwrjg_j32gdz6s542xlxy0m0037zm/T/testing/yield.js:63:6 <- /var/folders/nv/ycwrjg_j32gdz6s542xlxy0m0037zm/T/d6180c0ce24ad620fb39441b3b820153.browserify:76729:7)
```